src/sgame/sg_main.cpp

```diff
@@ -2273,6 +2276,10 @@ can see the last frag.
 */
 void CheckExitRules()
 {
```

Comment:
Would that work? I thought the same thing bug then I couldn't find any use of the Cvar class in sgame. If they do work, maybe it's worth stating so in the comment above the definition of vmCvar_t because it isn't very clear.